src/lib/drishti/CMakeLists.txt

```diff
@@ -328,6 +328,9 @@ endif()
 
 foreach(library ${drishti_libs} drishti drishtisdk)
   set_property(TARGET ${library} PROPERTY FOLDER "libs/drishti")
+  if(${DRISHTI_ADD_TO_STRING})
```

Comment:
Can be just `if(DRISHTI_ADD_TO_STRING)`.

Reply:
ok